### PR TITLE
[feature] Add load i18n files feature

### PR DESCRIPTION
**Pull request by ycs77**
This PR added load i18n files feature

### Changed

* Add load i18n files feature
* Add locales: `ru`, `zh-CN`, `zh-TW`
* Rename `example/example-i18n.html` to `example/example-i18n-customize.html`, and add new `example/example-i18n.html`
* New `create-locale` command

#### Use i18n from ES module

```js
import EditorJS from '@editorjs/editorjs';
import EditorJSLocaleZhTw from '@editorjs/editorjs/locales/zh-TW';

var editor = new EditorJS({
  // ...

  i18n: EditorJSLocaleZhTw
});
```

#### Use i18n from UMD (Browser)

```html
<script src="editorjs/editor.js"></script>
<script src="editorjs/locales/zh-TW.js"></script>

<script>
var editor = new EditorJS({
  // ...

  i18n: EditorJSLocaleZhTw
});
</script>
```

### Create new locale steps

1. Run `node scripts/create-locale en-US` (Change `en-US` to new locale name). This command will create `src/components/i18n/locales/en-US` folder (has base translation file) and `locales/en-US.d.ts`.
2. Translate `messages.json` file, no need to change `index.ts` file.
3. Run `yarn build` to build locale file.

### CODE REVIEW

**Comment by ycs77** [comment]
#751

File: src/components/i18n/locales/en/messages.json [hunk 1]
```diff
@@ -19,11 +19,27 @@
   },
   "toolNames": {
     "Text": "",
+    "Heading": "",
```
Comment (neSpecc) [inline src/components/i18n/locales/en/messages.json:22]: we cant hardcode tools list in the editor core

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: src/components/i18n/locales/en/messages.json (hunk 1 above)
Comment (ycs77) [reply src/components/i18n/locales/en/messages.json]: Removed

**Comment by ycs77** [comment]
Removed more translate key in "toolNames"

**Pull request closed without merging**